### PR TITLE
manual sync with upstream

### PR DESCRIPTION
**Pull request by trvswgnr**
let's get #65 finalized soon

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
+	url = https://github.com/crablang/edition-guide.git
+    branch = master
 [submodule "src/llvm-project"]
 	path = src/llvm-project
 	url = https://github.com/rust-lang/llvm-project.git
 	branch = rustc/16.0-2023-06-05
 [submodule "src/doc/embedded-book"]
 	path = src/doc/embedded-book
-	url = https://github.com/rust-embedded/book.git
+	url = https://github.com/crablang/embedded-book.git
```
Comment (dreyfus92) [inline .gitmodules:39]: just wondering if this link shouldn't be named like this `https://github.com/crab-embedded/book.git`? what do you think?

**Review by dreyfus92 — COMMENTED** [review]
I'm not sure if you have a specific pattern for renaming stuff, but kinda following what you're doing, I just leave you a comment.

**Comment by asyncfn** [comment]
You can't cover up homophobic behavior in your team!
crab
crab2

File: .gitmodules (hunk 1 above)
Comment (trvswgnr) [reply .gitmodules:39]: we don't have crab-embedded org so it matches our org:repo structure

Comment (dreyfus92) [reply .gitmodules:39]: fair enough

**Pull request merged**